db/1.14/modules/ROOT/pages/database-connector-input-output-parameters.adoc

```diff
@@ -0,0 +1,219 @@
++
+[source,xml,linenums]
+----
+{
+        "name": "customerId",
+        "type": "INTEGER",
+        "value": #[flowVars.customerId]
+    },
+    {
+        "name": "orderStatus",
+        "type": "VARCHAR",
+        "value": #[message.inboundProperties['orderStatus']]
+    }
+----
+* `name`
++
+The name of the input parameter.
+* `type`
++
+The data type of the input parameter.
+* `value`
++
+The value of the input parameter, specified using Mule expression syntax.
+
+The configuration enables you to fetch order details based on the provided customer ID and order status dynamically.
+
+In the *Configuration XML* editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:stored-procedure config-ref="Database_Config" doc:name="Stored Procedure">
+    <db:sql>
+        <![CDATA[{call GetOrderDetails(:customerId, :orderStatus)}]]>
+    </db:sql>
+    <db:input-parameters>
+        <![CDATA[#[{
+            "name": "customerId",
+            "type": "INTEGER",
+            "value": #[flowVars.customerId]
+        },
+        {
+            "name": "orderStatus",
+            "type": "VARCHAR",
+            "value": #[message.inboundProperties['orderStatus']]
+        }]]]>
+    </db:input-parameters>
+</db:stored-procedure>
+----
+
+== Configure Output Parameters
+
+Consider a scenario, where you develop a Mule app for an online retail system. The Mule app manages order processing, and you want to implement a feature to retrieve order details and calculate the total amount of an order from a MySQL database using a stored procedure. The stored procedure returns details such as the order items and quantities, and the total amount as an output parameter. The following example demonstrates how to configure the *Output parameters* field of the *Stored procedure* operation to fetch order details and the total amount dynamically.
+
+. In Studio, add a Database Configuration element to your MuleSoft app project and configure it with the necessary MySQL database connection details.
+. Drag the *Stored procedure* operation to your canvas that retrieves the order information.
+. Select the operation from your flow.
+. In the *SQL Query text* field, enter `{call GetOrderDetails(:orderId, :totalAmount, :orderDetails)}`.
++
+This defines the SQL statement for calling the stored procedure `GetOrderDetails` retrieving output parameters such as `orderId`, `totalAmount`, and `orderDetails`.
++
+. Set *Output parameters* to *Edit inline*.
+. Click the plus icon to open the Output parameter window.
+. In the *Output parameter* window, set the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Key | Type
+|orderId | INTEGER
+|totalAmount | Decimal
+|orderDetails | ARRAY
+|===
++
+[start=7]
+. Click *Finish*.
+
+The configuration enables you to execute the stored procedure, retrieving output parameters such as the order ID, total amount, and order details.
```

Comment:
```suggestion
The configuration enables you to execute the stored procedure to retrieve output parameters such as the order ID, total amount, and order details.
```

Reply:
I don't think that you need this paragraph. It is repetitive with the lead-in to the example.